using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -626,18 +712,20 @@ Identity.getExperienceCloudId(new AdobeCallback<String>() {
 {% endtab %}
 
 {% tab title="iOS" %}
-### getExperienceCloudId
+### getExperienceCloudId<a id="getExperienceCloudId-ios"></a>
 
-Retrieves the Adobe Experience Cloud Visitor ID from the Adobe Experience Cloud ID Service.
+This `getExperienceCloudId` API retrieves the Experience Cloud ID that was generated at initial launch  and stored in the Experience Cloud ID Service.
```

Comment:
- Update to 
"This API retrieves the Experience Cloud ID that was generated when the app was initially launched and is stored in the Experience Cloud ID Service."